Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyRowWidget.cpp

```diff
@@ -1744,11 +1751,14 @@ namespace AzToolsFramework
         {
             return false;
         }
-
-        return m_parentRow->CanChildrenBeReordered();
+        if (m_parentRow->m_reorderAllow)
+        {
+            return m_parentRow->CanChildrenBeReordered();
+        }
+        return false;
```

Comment:
Can this be reduced to just the following?

`return m_parentRow->m_reorderAllow && m_parentRow->CanChildrenBeReordered();`

Reply:
Done.